package.json

```diff
@@ -150,6 +150,7 @@
     "reselect": "^4.0.0",
     "rimraf": "^3.0.2",
     "source-map-support": "^0.5.16",
+    "styled-components": "^5.1.0",
```

Comment:
💅🏼

I think this is such a handy, well-made library, let's just start using it.

Reply:
https://styled-components.com/